tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
+
+    global log_stream
```

Comment:
I try to comply to PEP-8, but I see, that there are some useless blank lines. I'll clear that up.